### PR TITLE
Provides optional types for TypeScript

### PR DESCRIPTION
**Pull request by qrac**
Hi! I am using rehype-prism in TypeScript and it would be helpful if a type of plugin list is provided.

The editor completion works by exporting `Options`. No inconvenience associated with this. Please merge if you like.

ScreenShot 2022-03-20 17 15 27

**Please check if the PR fulfills these requirements**

- [x] Have you followed the guidelines in our [Contributing document](https://github.com/Val-istar-Guo/rehype-prism/blob/master/.github/CODE_OF_CONDUCT.md)?
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you written or modified docs for your core changes, as applicable?

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ const visitor = (preNode: hast.Element): void => {
 const selector = (node: hast.Element): boolean => node.tagName === 'pre'
 
 
-interface Options {
+export interface Options {
```
Comment (Val-istar-Guo) [inline src/index.ts:46]: It is better to rename Options to RehypePrismOptions, which can avoid naming conflicts as much as possible, if we export the interface.

**Review by Val-istar-Guo — CHANGES_REQUESTED** [review]

**Comment by Val-istar-Guo** [comment]
It's great to see someone maintain this repository with me ^_^

File: src/index.ts (hunk 1 above)
Comment (qrac) [reply src/index.ts]: Makes sense! Corrected.

**Review by Val-istar-Guo — APPROVED** [review]

**Comment by Val-istar-Guo** [comment]
new version released https://github.com/Val-istar-Guo/rehype-prism/pull/13

**Comment by qrac** [comment]
Thank you!

**Pull request merged**